githubflow_release/run.py

```diff
@@ -0,0 +1,54 @@
+"""Github flow release
+
+Usage:
+  githubflow_release (--release-type <type>)
+                     (--github-repo <repo>)
+                     [--defaults-file=FILE]
+                     [--project-path=DIR]
+                     [--remote-name=<name>]
+                     [--github-user=<user>]
+                     [--github-token=<token>]
+                     [--base-branch=<branch>]
+                     [--debian-changelog]
+                     [--excluded-pr-tag <tags>]...
+                     [--hotfix-pr-id <pr-id>]...
+  githubflow_release (-h | --help)
+  githubflow_release --version
+
+Options:
+  -h --help                 Show this screen.
+  --version                 Show version.
+  --defaults-file=FILE      Defaults file  [default: gitflow_release.yml]
+  --project-path=DIR        Project path   [default: .]
+  --release-type=<type>     Release type   [default: minor]
+  --remote-name=<name>      Remote name    [default: upstream]
+  --github-repo=<repo>      Github repo
+  --github-user=<user>      Github user
+  --github-token=<token>    Github toke
+  --base-branch=<branch>    Base branch  [default: master]
+  --debian-changelog        Generate debian_changelog
+  --excluded-pr-tag=<tags>  PR will be excluded if labelled with the given tag [default: hotfix not_in_changelog]
+  --hotfix-pr-id=<pr-id>      Hotfix PR ID
+"""
+from docopt import docopt
+from githubflow_release.release import release
+
+
+def main():
+    arguments = docopt(__doc__, version='Github flow 1.0.0')
+
+    release(defaults_file=arguments['--defaults-file'],
```

Comment:
I think this parameter is now useless, if I remember correctly, it was handled by clingon to to override cli params with a yml file. It was a nice feature, but I agree that it is not worth keeping clingon for this

Reply:
Another reason that I removed clingon is that clingon is not python3.6-friendly... I don't want to pass too much time on clingon because I'm a lazy guy :D